README.md

```diff
@@ -180,6 +180,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Apply templates with Jinja2](https://github.com/cuchi/jinja2-action) - Use the Jinja2 template engine to generate files from templates.
 - [Has Changes](https://github.com/UnicornGlobal/has-changes-action) - Check if there are code changes from previous steps.
 - [Mind Your Language Action](https://github.com/tailaiw/mind-your-language-action) - Detect offensive comments in issues and pull requests, and warn senders.
+- [YAML/JSON/XML Converter](https://github.com/fabasoad/yaml-json-xml-converter-action) - Takes file that in YAML/JSON/XML format and produces it in output in YAML/JSON/XML format.
```

Comment:
I would change this appended description to something more succinct:
"Converts YAML/JSON/XML file formats interchangeably. "

Reply:
Done. Please check.

Reply:
Not sure about the reason of Lint failures... It says that 185 links in this README file are dead but they are not. Sounds like Lint is broken. Could you please take a look?

Reply:
Looks good. Please pull master and push up again. That lint check is temporarily disabled until a more robust solution is in place.